pylint/checkers/classes.py

```diff
@@ -133,6 +133,37 @@ def _positional_parameters(method):
         positional = positional[1:]
     return positional
 
+def _has_different_parameters_default_value(original, overridden):
+    """
+    Check if original and overridden methods arguments have the same default values
+
+    Return True if one of the overridden arguments has a default
+    value different from the default value of the original argument
+    If one of the method doesn't have argument (.args is None)
+    return False
+    """
+    if original.args is not None and overridden.args is not None:
```

Comment:
You can invert condition in if, and do early return. It'll help with reducing indentation in code below